### PR TITLE
done with the product landing page

### PR DESCRIPTION
**Pull request by sohan9819**
not done with the responsiveness

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,7 +4,79 @@
...
+            id="search__box"
+            placeholder="Search"
+            class="ml-1 py-1 rounded-s nav__search-box color-black"
+          />
+        </i>        
+            
+        <div class="col-1-0f-4 p-1 text-center rounded-s h4 flex justify-evenly items-center gap-2">
+          <a href="#" class="uppercase h5 btn--small rounded-s bg-alpha btn__alpha-contained color-white">Login</a>
+          <i class="heart__icon fas fa-heart color-grey badge__basic-icon-m "></i>
+          <span class="cart__icon__wrapper"><i class="cart__icon fas fa-shopping-cart color-grey badge__basic-icon-m "></i><h4 class="cart__icon__text h5 pl-1 color-grey">&nbsp;Cart</h4></span>
```
Comment (sakshi006) [inline index.html]: Badges must have numbers on them to indicate the items in cart/wishlist.

**Review by sakshi006 — COMMENTED** [review]
Commented some changes.
Rest looks good.

**Comment by sohan9819** [comment]
> Commented some changes.
> Rest looks good.

Thanks for pointing that out I didn't noticed about that . Will work on it  🙂.

**Review by masraniJay — APPROVED** [review]
Looks good.

**Review by Rutwik87 — APPROVED** [review]
LGTM

File: index.html (hunk 1 above)
Comment (sudosohan) [reply index.html]: fixed it 👍

**Pull request merged**